nautobot_ssot/navigation.py

```diff
@@ -1,22 +1,29 @@
 """Plugin additions to the Nautobot navigation menu."""
 
-from nautobot.extras.plugins import PluginMenuItem
+from nautobot.apps.ui import NavMenuGroup, NavMenuItem, NavMenuTab
 
 
-menu_items = (
-    PluginMenuItem(
+items = [
+    NavMenuItem(
         link="plugins:nautobot_ssot:dashboard",
-        link_text="Dashboard",
+        name="Dashboard",
         permissions=["nautobot_ssot.view_sync"],
     ),
-    PluginMenuItem(
+    NavMenuItem(
         link="plugins:nautobot_ssot:sync_list",
-        link_text="History",
+        name="History",
         permissions=["nautobot_ssot.view_sync"],
     ),
-    PluginMenuItem(
+    NavMenuItem(
         link="plugins:nautobot_ssot:synclogentry_list",
-        link_text="Logs",
+        name="Logs",
         permissions=["nautobot_ssot.view_synclogentry"],
     ),
+]
+
+menu_items = (
+    NavMenuTab(
+        name="Automation",
```

Comment:
Is this an existing one?

Reply:
Yes it is, and it's what I understand we're supposed to be putting Apps under.

Reply:
For 2.0 we're keeping the existing menu structure for the legacy UI - see https://github.com/nautobot/nautobot/pull/3964, this is a backtrack from what was present in beta 1.